business/data-confidentiality.md

```diff
@@ -0,0 +1,15 @@
+---
+description: Data confidentiality in machine translation
+---
+
+# Data confidentiality
+
+**Data confidentiality** refers to how the machine translation providers use the specific data sent for translation. Providers can generate translations with or without storing the input. Machine translation providers need data to train and improve their translation service.
+
+In systems with no confidentiality settings, input data can reappear in similar machine translation calls. Providers that offer no-trace never store data.
```

Comment:
> input data can reappear in similar machine translation calls

This is an important point.  As usual, I tend towards structuring this, because this is a very different type of risk.

> ### Risk types
> Confidential data can be exposed when:
> - Training data is accessed.
> - Request data is accessed.
> - Training data is included as part of translation output.